packages/components/src/Form/Inputs/InputFilters/InputFilters.tsx

```diff
@@ -119,102 +119,123 @@ const InputFiltersLayout: FC<InputFiltersProps> = ({
+          )
+          return filter.field === fieldEditing ? (
+            <Popover
+              key={i}
+              isOpen={fieldEditing !== undefined}
+              setOpen={closeInputFiltersChipEditor}
+              content={
+                <InputFiltersChipEditor
+                  value={filter.value}
+                  onChange={setFieldEditingValue}
+                  options={filter.options}
+                />
+              }
+            >
+              {filterToken}
+            </Popover>
+          ) : (
+            filterToken
+          )
+        })}
+        {!fieldEditing && (
+          <Select
+            autoResize
+            openOnFocus
+            indicator={false}
+            onFilter={setFilterTerm}
+            isFilterable
+            onChange={handleFilterLookupChange}
+            options={options}
+            placeholder="Filter List"
+            ref={inputRef}
           />
-        ) : (
-          <Text fontSize="small">{filter?.label || filter.field}</Text>
-        )
-        return filter.field === fieldEditing ? (
-          <Popover
-            key={i}
-            isOpen={fieldEditing !== undefined}
-            setOpen={closeInputFiltersChipEditor}
-            content={
-              <InputFiltersChipEditor
-                value={filter.value}
-                onChange={setFieldEditingValue}
-                options={filter.options}
-              />
-            }
-          >
-            {filterToken}
-          </Popover>
-        ) : (
-          filterToken
-        )
-      })}
-
-      {!fieldEditing && (
-        <Select
-          autoResize
-          openOnFocus
-          indicator={false}
-          onFilter={setFilterTerm}
-          isFilterable
-          onChange={handleFilterLookupChange}
-          options={options}
-          placeholder="Filter List"
-          ref={inputRef}
-        />
-      )}
+        )}
+      </ChipWrapper>
       {isClearable && (
         <IconButton
           icon="Close"
           label="Clear Filters"
+          ml="auto"
           mr="xxsmall"
+          mt="xxsmall"
           onClick={clearFilters}
           size="xsmall"
-          ml="auto"
         />
       )}
     </div>
   )
 }
 
+const ChipWrapper = styled.div`
+  display: inline-flex;
+  flex: 1;
+  flex-wrap: wrap;
+
+  ${Chip} {
+    margin-right: ${({ theme: { space } }) => space.xsmall};
+  }
+
+  @supports (gap: 4px) {
+    gap: ${({ theme }) => theme.space.xsmall};
+    ${Chip} {
+      margin: 0;
+    }
+  }
+`
+
 export const InputFilters = styled(InputFiltersLayout)`
-  align-items: center;
+  align-items: start;
   border: solid 1px ${({ theme }) => theme.colors.ui2};
   border-radius: ${({ theme: { radii } }) => radii.medium};
   display: flex;
-  height: 36px;
+  flex-wrap: wrap;
+  min-height: 36px;
   padding: ${({ theme: { space } }) => `${space.xxxsmall} ${space.xxsmall}`};
+  padding-top: ${({ theme: { space } }) => `${space.xsmall}`};
```

Comment:
With the removal of the `div` around the `Chip` you'll also need to change the padding here and add `lineHeight="xlarge"` to the `Text`.

```suggestion
  padding: ${({ theme: { space } }) => `${space.xxxsmall} ${space.xxsmall}`}
```